commands/daemonize/daemonize.go

```diff
@@ -0,0 +1,70 @@
+package daemonize
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/takama/daemon"
+	cmd_ep "github.com/taskcluster/taskcluster-worker/commands/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+const (
+	// name of the service
+	name        = "taskcluster-worker"
+	description = "Taskcluster worker"
+)
+
+var dependencies = []string{}
+
+func init() {
+	cmd_ep.CommandProviders.Register(cmd{}, "daemon")
```

Comment:
nit: the package name is `daemonize`, the directory is `daemonize`, the file is `daemonize.go` but the command is `daemon`. I think they should all be one or the other for consistency. I see we import a package called `daemon` but we could import it as something else if needed.